tests/containers/docker_image.pm

```diff
@@ -35,11 +35,6 @@ sub run {
 
     install_docker_when_needed();
 
-    if (is_sle()) {
-        ensure_ca_certificates_suse_installed();
-        allow_selected_insecure_registries(runtime => 'docker');
```

Comment:
Why do you remove this line? The `tests/containers/import_ca_certs.pm` is not covering that.

Reply:
nice catch. that s my bad because i thought that the setup is happening prior to the `docker_image.pm` in the `podman_image.pm`. surprisingly the VR passed and i totally missed it. thanks